PLUGINS.md

```diff
@@ -0,0 +1,89 @@
+| MAX_GAS_PER_BLOCK       | 30_000_000L | `--plugin-linea-max-block-gas` |
+
+### Transaction validation - LineaTransactionValidatorPlugin
+
+This plugin extends the default transaction validation rules for adding transactions to the
+transaction pool. It leverages the PluginTransactionValidatorService to manage and customize the
+process of transaction validation. This includes, for example, setting a deny list of addresses
+that are not allowed to add transactions to the pool.
+
+#### CLI Options
+
+| Option Name | Default Value | Command Line Argument |
+| --- | --- | --- |
+| DENY_LIST_PATH | lineaDenyList.txt | `--plugin-linea-deny-list-path` |
+| MAX_TX_GAS_LIMIT_OPTION | 30_000_000 | `--plugin-linea-max-tx-gas-limit` |
+| MAX_TX_CALLDATA_SIZE | 60_000 | `--plugin-linea-max-tx-calldata-size` |
+
+## RPC
+
+### Counters - RollupCounterRpcPlugin
+#### `rollup_getTracesCountersByBlockNumberV0` 
+
+The RollupRpcEndpointServicePlugin registers an RPC endpoint named `getTracesCountersByBlockNumberV0` 
+under the `rollup` namespace. When this endpoint is called, returns trace counters based on the provided request parameters.
+
+#### Parameters
+
+  - `blockNumber`: _string_ - The block number
+
+  - `tracerVersion`: _string_ - The tracer version. It will return an error if the 
+requested version is different from the tracer runtime 
+
+### Trace generation - RollupTraceGenerationRpcPlugin
+
+This plugin registers an RPC endpoint named `generateConflatedTracesToFileV0` under the `rollup` namespace. 
+The endpoint generates conflated file traces.
+
+#### Parameters
+
+- `fromBlock`: _string_ - the fromBlock number
+- `toBlock`: _string_ - The toBlock number
+- `tracerVersion`: _string_ - The tracer version. It will return an error if the
+  requested version is different from the tracer runtime
+
+## Continuous Tracing
+
+The continuous tracing plugin allows to trace every newly imported block and use Corset to check if the constraints are
+valid. In case of an error a message will be sent to the configured Slack channel.
+
+### Usage
+
+The continuous tracing plugin is disabled by default. To enable it, use the `--plugin-linea-continuous-tracing-enabled`
+flag. If the plugin is enabled it is mandatory to specify the location of `zkevm.bin` using
+the `--plugin-linea-continuous-tracing-zkevm-bin` flag. The user with which the node is running needs to have the
+appropriate permissions to access `zkevm.bin`.
+
+In order to send a message to Slack a webhook URL needs to be specified by setting the `SLACK_SHADOW_NODE_WEBHOOK_URL`
+environment variable. An environment variable was chosen instead of a command line flag to avoid leaking the webhook URL
+in the process list.
+
+The environment variable can be set via systemd using the following command:
+
+```shell
+Environment=SLACK_SHADOW_NODE_WEBHOOK_URL=https://hooks.slack.com/services/SECRET_VALUES
+```
+
+### Invalid trace handling
```

Comment:
You define what happens for invalid traces, but nothing about what happens if there are no problems.

Reply:
that is in the first line of that paragraph, in case of success the trace file is just deleted